### ISSUE TITLE
Home Assistant nodes version 0.57.0+

### ISSUE THREAD

**Issue by cob94440** [zachowj/hass-node-red#214]
hello,
I only have 0.56.1 on my nodered as 0.57 is only in beta. if i update Node-RED Companion it wont work until 0.57 is out or i update to beta correct?

**Comment by zachowj** [comment]
Correct, but 3.0.0 will work with 0.56.1 if you're not using any device nodes.
